src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/on--event--.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'on(event)'
-excerpt: 'Browser module: Browser.on method'
+excerpt: 'Browser module: on method'
```

Comment:
This method was removed in https://github.com/grafana/k6-docs/pull/1197. Be aware of conflicts.

Reply:
See: 2d2d4076ba9c8f7e9cba7540b537b8d54c1d9eed